Modules/itertoolsmodule.c

```diff
@@ -4236,8 +3762,17 @@ repeat_repr(repeatobject *ro)
                                     ro->cnt);
 }
 
+/*[clinic input]
+itertools.repeat.__length_hint__
+
+    ro: self(type="repeatobject *")
+
+Private method returning an estimate of len(list(it)).
```

Comment:
Aren’t _ro_ and _it_ the same parameter?

Reply:
Yes, but `ro` isn't included in the doc-string and would be unclear to users IMO. Suggestion for something more informative than `it` are welcome!

Currently this gives:
```
>>> help(itertools.repeat.__length_hint__)
Help on method_descriptor:

__length_hint__(self, /)
    Private method returning an estimate of len(list(it)).
```

Reply:
Why not use `self`?